tests/metaworld/envs/mujoco/sawyer_xyz/test_scripted_policies.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+from metaworld.envs.mujoco.env_dict import ALL_ENVIRONMENTS
+from metaworld.policies import *
+from tests.metaworld.envs.mujoco.sawyer_xyz.utils import check_success
+
+
+test_data = [
+    # name,      policy,      success rate,   environment kwargs
+    ['door-open-v1', SawyerDoorOpenV1Policy(), 0.95, {}],
+    ['door-close-v1', SawyerDoorCloseV1Policy(), 0.95, {}],
+    # Expected success is low because noise has huge negative impact on drawer-open
+    ['drawer-open-v1', SawyerDrawerOpenV1Policy(), 0.50, {}],
+    ['drawer-close-v1', SawyerDrawerCloseV1Policy(), 0.84, {}]
+]
+
+for row in test_data:
+    row[-1] = ALL_ENVIRONMENTS[row[0]](random_init=True, **row[-1])
```

Comment:
can you explain what's going on here?

Reply:
It's just replacing the kwargs of the list with an environment that's been instantiated using said kwargs.